application/src/main/data/upgrade/3.4.4/schema_update.sql

```diff
@@ -89,6 +89,77 @@ CREATE TABLE IF NOT EXISTS alarm_comment (
 ) PARTITION BY RANGE (created_time);
 CREATE INDEX IF NOT EXISTS idx_alarm_comment_alarm_id ON alarm_comment(alarm_id);
 
+CREATE TABLE IF NOT EXISTS notification_target (
+    id UUID NOT NULL CONSTRAINT notification_target_pkey PRIMARY KEY,
+    created_time BIGINT NOT NULL,
+    tenant_id UUID NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    configuration VARCHAR(10000) NOT NULL,
+    CONSTRAINT uq_notification_target_name UNIQUE (tenant_id, name)
```

Comment:
Add an (ON DELETE CASCADE) constraint to delete all this data if we delete the tenant. See alarm_comment for example. Is this for storing the recipients?

Reply:
We need to store these entities on sysadmin level, and with constraint we cannot use NULL_UUID.
But if we use null for tenant_id for entities on sysadmin level, then we will face search issues, as we will need to insert smth like that in each query:  :tenantId is null and tenant_id is null or :tenantId = tenant_id